_redirects

```diff
@@ -6,3 +6,25 @@ layout: null
 /repl/build/:build /7/ 200
 # Previous versions of the REPL
 /repl/version/:version /7/ 200
+
+# Ported from redirect_from
+
+/docs/using-babel/                  /docs/setup/
```

Comment:
I was going to say "make these 301 (permanent) redirects", but Netlify uses 301 by default: https://www.netlify.com/docs/redirects/#http-status-codes. Nice.